vulture_os/applications/templates/apps/logfwd.html

```diff
@@ -62,24 +63,47 @@ <h1 class="panel-title"><i class="fa fa-sitemap">&nbsp;</i>{% translate "Logs Fo
       name: "internal",
       aTargets: [2],
       defaultContent: "",
-      mData: "internal"
+      mData: "internal",
+      bVisible: false
+    },
+    {
+      sTitle: '{% translate "Enabled" %}',
```

Comment:
Is it really needed to add another column in the GUI.
Can we move the status next to the name as it is done in Nodes, Frontend and Backend views ?

Reply:
Solved : Name column now holds the information of enabled/disabled forwarders